### PR TITLE
fix: improve feedback form

### LINKED ISSUE #3974: Improve feedback form when users are rating the tool
### Is your enhancement related to a problem? Please describe

Podman Desktop provides a way for the user to share their feedback. Users can also share a rating for the application.
When a user is providing a rate of 1 (on 4), we are sometime missing context which would be valuable to help improving the application. 

### Describe the solution you'd like

When a user is rating with 1 the application, we should nag the user to provide us actionnable feedback:
- Require the user to provide a message to explain a particular issue
- Re-inforce that we are taking the message seriously and we would like to get in touch with the user to investigate a potential issue.

### Describe alternatives you've considered

- Extend the feedback form to add more fields to better identify the reason the user is not satisfied
- Make email and textfield mandatory when rating is 1 

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

If you select the unhappy smiley and haven't added details or contact info, the feedback button stays disabled and there's a new error message: "Please share contact info or details on how we can improve".

If you select the 'meh' smiley and haven't entered details the feedback is not disabled, but there's a new warning message: "We would really appreciate knowing how we can improve".

The form also has a new placeholder in the details to say we appreciate and review all comments.

Suggestions on improving these feedback messages appreciated. :)

Two minor indirect fixes: had to fix the bottom layout to handle longer messages, and updated WarningMessage to match ErrorMessage (it was missing vertical alignment and aria fixes that ErrorMessage had).

### Screenshot/screencast of this PR

Screenshot 2023-09-28 at 3 33 33 PM
Screenshot 2023-09-28 at 3 33 44 PM

### What issues does this PR fix or reference?

Fixes #3974.

### How to test this PR?

Open feedback form and try different smileys.

### CODE REVIEW

**Review by jeffmaury — CHANGES_REQUESTED** [review]
I noticed that if we enter space(s) in description or contact info then the form can be submitted. We should fix that (as it's my first try when some stuff force me to specify information)

**Comment by deboer-tim** [comment]
> I noticed that if we enter space(s) in description or contact info then the form can be submitted. We should fix that (as it's my first try when some stuff force me to specify information)

I can keep it disabled for whitespace, but I just type `blah` or `bob@nowhere.com` if a form makes me do that, not sure we'll actually get a higher response rate.

**Comment by jeffmaury** [comment]
> > I noticed that if we enter space(s) in description or contact info then the form can be submitted. We should fix that (as it's my first try when some stuff force me to specify information)
> 
> I can keep it disabled for whitespace, but I just type `blah` or `bob@nowhere.com` if a form makes me do that, not sure we'll actually get a higher response rate.

I aggree it's not an absolute protection but it still improves

**Review by benoitf — APPROVED** [review]

**Comment by deboer-tim** [comment]
> > I can keep it disabled for whitespace, but I just type `blah` or `bob@nowhere.com` if a form makes me do that, not sure we'll actually get a higher response rate.
> 
> I aggree it's not an absolute protection but it still improves

See latest commit - I defined 'valid feedback' as at least 5 characters after trimming in either the text area or contact info. Made the sad smiley use the same path for simplicity.

File: packages/renderer/src/lib/feedback/SendFeedback.svelte [hunk 1]
```diff
@@ -110,17 +116,27 @@ async function sendFeedback(): Promise<void> {
           name="contactInformation"
           id="contactInformation"
           bind:value="{contactInformation}"
-          placeholder="Enter email address, phone number or leave blank for anonymous feedback"
+          placeholder="Enter email address, phone number, or leave blank for anonymous feedback"
```
Comment (jeffmaury) [inline packages/renderer/src/lib/feedback/SendFeedback.svelte]: As it's very unlikely that user will give their phone number, can we switch to type=email and update placeholder

**Review by jeffmaury — APPROVED** [review]
LGTM

File: packages/renderer/src/lib/feedback/SendFeedback.svelte (hunk 1 above)
Comment (deboer-tim) [reply packages/renderer/src/lib/feedback/SendFeedback.svelte]: Updated.

**Pull request merged**